docker-compose.nodb.yml

```
@@ -0,0 +1,107 @@
+services:
+
+    # Frontend - React + nginx
+    frontend:
+        image: buildingsmart/validationsvc-frontend:${VERSION}
+        build:
+            context: .
+            dockerfile: ./docker/frontend/Dockerfile
+            target: run
+        restart: unless-stopped
+        container_name: frontend
+        ports:
+            - 80:80
+            - 443:443
+        volumes:
+            - ./docker/frontend/letsencrypt:/etc/letsencrypt
+            - static_data:/app/backend/django_static
+        depends_on:
+            - backend
+
+    # Backend - Django
+    backend:
+        image: buildingsmart/validationsvc-backend:${VERSION}
+        build:
+            context: .
+            dockerfile: ./docker/backend/Dockerfile
+            target: run
+        entrypoint: /app/backend/server-entrypoint.sh
+        restart: unless-stopped
+        container_name: backend
+        volumes:
+            - static_data:/app/backend/django_static
+            - files_data:/files_storage
+        ports:
+            - 8000:8000
+        environment:
+            PUBLIC_URL: ${PUBLIC_URL}
+            DEBUG: ${DEBUG}
+            CELERY_BROKER_URL: ${CELERY_BROKER_URL}
+            CELERY_RESULT_BACKEND: "django-db"
+            CELERY_RESULT_BACKEND_DB: "db+postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db/${POSTGRES_NAME}"
+            DJANGO_DB: ${DJANGO_DB}
+            DJANGO_SECRET_KEY: ${DJANGO_SECRET_KEY}
+            DJANGO_ALLOWED_HOSTS: ${DJANGO_ALLOWED_HOSTS}
+            DJANGO_TRUSTED_ORIGINS: ${DJANGO_TRUSTED_ORIGINS}
+            DJANGO_GUNICORN_WORKERS: ${DJANGO_GUNICORN_WORKERS}
+            DJANGO_GUNICORN_THREADS_PER_WORKER: ${DJANGO_GUNICORN_THREADS_PER_WORKER}
+            POSTGRES_HOST: ${POSTGRES_HOST}
+            POSTGRES_PORT: ${POSTGRES_PORT}
+            POSTGRES_NAME: ${POSTGRES_NAME}
+            POSTGRES_USER: ${POSTGRES_USER}
+            POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+            B2C_CLIENT_ID: ${B2C_CLIENT_ID}

```

```suggestion
            B2C_CLIENT_ID: ${B2C_CLIENT_ID}
            POSTGREST_EXTERNAL: 1
```

I think what we should do is add an env var here.. so that we can pick this up later in the entry point. I think running with integrated DB should remain working as it is now so I'd like to keep the functionality of waiting for the DB to come up.

---

Reverted db readiness checks, see no reason why we cannot check for external DB to be ready too? Agree?

---

I guess... If it's down you'd rather have a specific message rather than wait indefinitely though. But we can refine that later, it's a corner case.